docs/resources/aws_ecs_service.md

```diff
@@ -0,0 +1,82 @@
+---
+title: About the aws_ecs_service Resource
+platform: aws
+---
+
+# aws\_ecs\_service
+
+Use the `aws_ecs_service` InSpec audit resource to test properties of a single specific Athena Work Group.
+
+The AWS::ECS::Service resource creates an Amazon Elastic Container Service (Amazon ECS) service that runs and maintains the requested number of tasks and associated load balancers.
+
+## Syntax
+
+Ensure that a services exists.
+    describe aws_ecs_service(cluster: "default", service: ["arn:aws:ecs:us-east-1:012345678910:service/ecs-simple-service"]) do
```

Comment:
Will this take multiple services? Also does it accept the service arn and service name?